app/src/main/java/com/duckduckgo/app/cta/ui/CtaViewModel.kt

```diff
@@ -252,8 +293,76 @@ class CtaViewModel @Inject constructor(
                 }
             }
 
-            canShowPrivacyProCta() && extendedOnboardingFeatureToggles.privacyProCta().isEnabled() -> {
-                DaxBubbleCta.DaxPrivacyProCta(onboardingStore, appInstallStore)
+            canShowPrivacyProCta() -> {
+                if (highlightsOnboardingExperimentManager.isHighlightsEnabled()) {
```

Comment:
I thought we were gonna wait for this experiment to finish?

Reply:
We won’t run both experiments simultaneously but will enable this one as soon as the Highlights experiment reaches its sample size. To avoid delays, we need to release the code now so it’s ready when Highlights is finished.

If we wait for Highlights to complete, analyse the data, and then implement the copy test, it will significantly delay this experiment. Although I had to make some adjustments to integrate this test with Highlights, it’s worth it to speed up delivery